### ISSUE TITLE
No quota left and almost no keys left

### ISSUE THREAD

**Issue by eeeeeeeeeewastaken** [Benjamin-Loison/YouTube-operational-API#216]
I keep getting this error: "The request cannot be completed because the YouTube operational API run out of quota. Please try again later."
Website says there's only 1 key left? A few days ago it said 300 or so

**Comment by Benjamin-Loison** [comment]
I am currently investigating the issue: https://discord.com/channels/933841502155706418/933841503103627316/1186027752390922281

**Comment by Benjamin-Loison** [comment]
**TLDR: The issue seems to be gone and if it comes back I will be notified automatically in the minute. I will open back this issue if the issue arises again. Sorry for the interruption of service.**

As discussed in #11 and #165, I cannot guarantee SLA for the free official instances, hence I am not notified when the official instance is not reachable (due to too many requests sent to it), however when the official instance is reachable I want that it does not send incorrect results (note that an incorrect result means YouTube Data API v3 complaining about one of the key of our keys, hence not returning us the desired result to our request, note that being the no-key endpoint does not matter here even if it is in theory more stable than our webscraping endpoints), hence I enabled (and tested) sending me automatically a notification (over Matrix) in the minute (thanks to a crontab) an incorrect result is sent to a verification request.
